test/integration/smoke/test_service_offerings.py

```diff
@@ -811,8 +817,28 @@ def test_06_disk_offering_strictness_false(self):
             "Check service offering of the VM"
         )
 
+        if fullClone[0].value.lower() == "false":
+            self.updateVmwareSettings(True)
+
         return
 
+    def updateVmwareSettings(self, tearDown):
```

Comment:
send the value instead of tearDown

```suggestion
    def updateVmwareCreateFullCloneSetting(self, value):
```